tensorboard/compat/tensorflow_stub/io/gfile.py

```diff
@@ -386,8 +397,8 @@ def __init__(self, filename, mode):
         self.fs_supports_append = hasattr(self.fs, 'append')
         self.buff_chunk_size = _DEFAULT_BLOCK_SIZE
         self.buff = None
-        self.buff_offset = 0
-        self.offset = 0
+        self.buff_char_offset = 0
```

Comment:
Similar to the above, I think we want the buff offset to be measured in either bytes or chars depending on whether we're in binary mode, rather than always chars.

Reply:
done